src/main/java/com/epam/esm/integration/errorhandle/RestControllerExceptionHandler.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.esm.integration.errorhandle;
```

Comment:
Integraion isn't the correct package for this. Move errorhandle package to the top layer, alongside other features (like tag)